test/src/test/java/org/corfudb/integration/CorfuReplicationSiteConfigIT.java

```diff
@@ -364,6 +364,7 @@ public void runSiteSwitch() throws Exception {
                assertThat(mapA1.containsKey(String.valueOf(i))).isTrue();
             }
 
+            System.out.print("\nTest Succeeds!!!");
```

Comment:
please remove this otherwise compilation will fail

Reply:
This is the test, the system.out.print is allover in the tests.